modin/core/execution/ray/common/utils.py

```diff
@@ -65,6 +68,19 @@ def initialize_ray(
         If not specified, ``modin.config.RayRedisPassword`` is used.
     """
     extra_init_kw = {"runtime_env": {"env_vars": {"__MODIN_AUTOIMPORT_PANDAS__": "1"}}}
+
+    # Set Modin logging envvars if not default (if set).
+    if LogMode.get() != LogMode._get_default():
```

Comment:
```suggestion
    if LogMode.get_value_source() != ValueSource.DEFAULT:
```
without using protected methods (you'd have to import `ValueSource` from `modin.config` first, obviously)

for all configuration variables